flaskfront/opgflaskfront/templates/component.html

```diff
@@ -0,0 +1,141 @@
+{% extends "base.html" %}
```

Comment:
Should this whole file be removed? If we're keeping it, can we modify the index page so that it points to these components and says something like "Here's a link to some components to give examples of how to use the Jinja templates"?

Reply:
This was legacy. Now removed